### DISCUSSION TITLE
direct model binding and hasOne relation

### DISCUSSION THREAD

**Discussion by akamasAT** [livewire/livewire#5263] [1. Help]
Hello there!
I'm stuck with binding a hasOne relation. The docs only show how to bind a hasMany relation.

I have a Customer Model and load additional data, everything fine and i can echo it.

`$this->customer->load('sepaMandat', 'paymentType');`
`{{ $customer->sepaMandat->iban }}`

so if some paymentType is set, extra input fields should be shown and wired up with the stored data.
I have rules set for them like this:

`'customer.sepaMandat.iban'              => 'sometimes|required',
 'customer.sepaMandat.account_holder'    => 'sometimes|required',`

in the blade I try to wire it up with

`wire:model="customer.sepaMandat.iban"`

this seems not to be the right way since no data is shown/wired - where is my mistake? Is this even possible?

**Comment by eafarooqi** [comment]
Hi, 
use underscore between multi word. I have the same problem, so i have done something like this and data is shown. 
```
'configuration.email_configuration.email' => 'required',
wire:model.defer="configuration.email_configuration.email"
```

so in you case may be like this 
```
customer.sepa_mandat.iban
```
and
```
wire:model="customer.sepa_mandat.iban"
```

this shows the data but saving the model with belongsTo or hasOne relation will not work with the save method.

**Comment by joshhanley** [comment]
@akamasAT and @eafarooqi yeah there is issues with multiword relations at the moment when using model binding. See this discussion #3803 for more details.
